diff --git a/content/public/browser/browser_thread.h b/content/public/browser/browser_thread.h
index 89022cbd031e3..3c925798d3502 100644
--- a/content/public/browser/browser_thread.h
+++ b/content/public/browser/browser_thread.h
@@ -30,9 +30,9 @@ class BrowserTaskTraits;
 // Use DCHECK_CURRENTLY_ON(BrowserThread::ID) to assert that a function can only
 // be called on the named BrowserThread.
 #define DCHECK_CURRENTLY_ON(thread_identifier)                      \
-  (DCHECK(::content::BrowserThread::CurrentlyOn(thread_identifier)) \
-   << ::content::BrowserThread::GetDCheckCurrentlyOnErrorMessage(   \
-          thread_identifier))
+//  (DCHECK(::content::BrowserThread::CurrentlyOn(thread_identifier)) \
+//   << ::content::BrowserThread::GetDCheckCurrentlyOnErrorMessage(   \
+//          thread_identifier))
 
 // The main entry point to post tasks to the UI thread. Tasks posted with the
 // same |traits| will run in posting order (i.e. according to the
